src/compiler/commandLineParser.ts

```diff
@@ -332,6 +332,11 @@ namespace ts {
             description: Diagnostics.Do_not_emit_use_strict_directives_in_module_output
         },
         {
+            name: "listEmittedFiles",
+            type: "boolean",
+            description: Diagnostics.List_emitted_files

```

Comment:
do not add the description to avoid having it in the help. most users do not care about this flag